data/story northern union.txt

```diff
@@ -139,9 +142,9 @@ mission "Meeting an Alpha 1"
+			`	"I'm sure you know already that we were created as super soldiers to be used by various factions, but the reason for that is because those in charge were getting reports of normal soldiers refusing orders.  Things such as leveling towns full of innocent bystanders, shooting down unarmed passenger ships, contaminating spaceports with radiation or bioweapons, that sort of thing. The human soldiers were apparently brought up with the idea that 'life was sacred' or something." His tone turns to what you can only assume is mockery. "And that 'only bad people' should be killed. And when faced with the flimsy excuses for why they were slaughtering these innocents, they tended to refuse like the cowards they were."`
 			`	The being calls over a waiter, and whispers his order, before continuing. There is an interesting smell in the air, which both turns your stomach and compels you to be silent.`
-			`	"So, where was I?" He pauses, "Ah yes, our origins. It was decided that there was a need for soldiers who were "emotionless," and well cheaper to maintain for increased combat efficiency.  Starting with your "basic" human, it was decided to focus on developing a streamlined combat being, one which would obey orders without question, and could be fielded without much overhead. Turns out your human frame is about the only useful thing about you in combat, however. The limits of genetic engineering at the time, however, seemed to be infinite. So they started taking samples from various bugs. Cockroaches, bacteria strains, grasshoppers. They set out to create the perfect beings, us. A perfect blend of strenght, agility, tactical prowess, radiation and bioweapon resistances, and natural built in and regenerating armor in the form of our exoskeleton. Capable of surviving and fighting for hours in empty space, or under water, or in most any environment, really. Only occasionally with extra gear."`
+			`	"So, where was I?" He pauses, "Ah yes, our origins. It was decided that there was a need for soldiers who were "emotionless," and, well, cheaper to maintain for increased combat efficiency. Starting with your "basic" human, it was decided to focus on developing a streamlined combat being, one which would obey orders without question, and could be fielded without much overhead. Turns out your human frame is about the only useful thing about you in combat, however. The limits of genetic engineering at the time, however, seemed to be infinite. So they started taking samples from various bugs. Cockroaches, bacteria strains, grasshoppers. They set out to create the perfect beings, us. A perfect blend of strengh, agility, tactical prowess, radiation and bioweapon resistances, and natural built in and regenerating armor in the form of our exoskeleton. Capable of surviving and fighting for hours in empty space, or under water, or in most any environment, really. Only occasionally with extra gear."`

```

Comment:
Change:

> Turns out your human frame is about the only useful thing about you in combat, however. The limits of genetic engineering at the time, however, seemed to be infinite. So they started taking samples from various bugs.

To:

> Turns out your human frame is the only useful thing about you in combat. However, the limits of genetic engineering at the time, seemed to be infinite, so they started taking samples from various bugs.

The double "about" in 1 sentence reads awkward, and 2 instances of "however" in 2 consecutive sentences even more so. Lastly, the last sentence in my quote reads better when part of the previous one, since starting a sentence with "so" is generally a thing to avoid if possible (same with words like "but").

Reply:
Change:

> Cockroaches, bacteria strains, grasshoppers. They set out to create the perfect beings, us.

To:

> Cockroaches, bacteria strains, grasshoppers, they set out to create the perfect beings, us.

To be honest, I'm not sure if this is the best "fix", but having a sentence which only consists of a few "species" words doesn't read that nice.
Alternatively, you could instead merge the first quoted sentence with the sentence before it instead, and leave the second quoted sentence alone.